src/chains.ts

```diff
@@ -19,28 +21,17 @@ const baseL2Addresses = {
   fallbackHandlerAddress: "0x017062a1dE2FE6b99BE3d9d37841FeD19F573804",
   createCallAddress: "0xB19D6FFc2182150F8Eb585b79D4ABcd7C5640A9d",
   signMessageLibAddress: "0x98FFBBF51bb33A056B08ddf711f289936AafF717",
+  safeSingletonAddress: "0x",
+  simulateTxAccessorAddress: "0x59AD6735bCd8152B84860Cb256dD9e96b85F69Da",
 };
 
-// sepolia is the same as base.
-const sepoliaAddresses = baseL2Addresses;
-
 export const contractNetworks: ContractNetworksConfig = {
```

Comment:
do we need this? can it be commented out?

Reply:
It'll be helpful to keep at least one example if we need it.